pkg/config/application_config.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/storage"
+	"github.com/nspcc-dev/neo-go/pkg/network/metrics"
+	"github.com/nspcc-dev/neo-go/pkg/rpc"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+)
+
+// ApplicationConfiguration config specific to the node.
+type ApplicationConfiguration struct {
+	LogPath           string                  `yaml:"LogPath"`
+	DBConfiguration   storage.DBConfiguration `yaml:"DBConfiguration"`
+	Address           string                  `yaml:"Address"`
+	NodePort          uint16                  `yaml:"NodePort"`
+	Relay             bool                    `yaml:"Relay"`
+	DialTimeout       time.Duration           `yaml:"DialTimeout"`
+	ProtoTickInterval time.Duration           `yaml:"ProtoTickInterval"`
+	PingInterval      time.Duration           `yaml:"PingInterval"`
+	PingTimeout       time.Duration           `yaml:"PingTimeout"`
+	MaxPeers          int                     `yaml:"MaxPeers"`
+	AttemptConnPeers  int                     `yaml:"AttemptConnPeers"`
+	MinPeers          int                     `yaml:"MinPeers"`
+	Prometheus        metrics.Config          `yaml:"Prometheus"`
+	Pprof             metrics.Config          `yaml:"Pprof"`
+	RPC               rpc.Config              `yaml:"RPC"`
+	UnlockWallet      wallet.Config           `yaml:"UnlockWallet"`
```

Comment:
Maybe it's a good time to sort these fields, it will simplify subsequent maintenance.